packages/plugins/xray/README.md

```diff
@@ -0,0 +1,23 @@
+## `@envelop/xray`
+
+This plugin adds aws xray trace segments for resolvers using [`aws-xray-sdk`](https://github.com/aws/aws-xray-sdk-node).
+
+## Getting Started
+
+```
```

Comment:
```suggestion
```sh
```

Reply:
Yeah, lots of stuff missing from the docs still, I hadn't really gotten to that yet, this is just a stripped down version of docs from another plugin